### PR TITLE
Upgrade JUnit Platform 1.1.0 / JUnit Jupiter 5.1.0

### PR DESCRIPTION
**Pull request by raniejade**

### CODE REVIEW

**Review by arturdryomov — APPROVED** [review]

File: gradle.properties [hunk 1]
```diff
@@ -1,5 +1,5 @@
 kotlinVersion=1.2.20
-junitPlatformVersion=1.0.1
+junitPlatformVersion=1.1.0
```
Comment (arturdryomov) [inline gradle.properties:2]: BTW WDYT about moving all versions to a single place?

Comment (artem-zinnatullin) [reply gradle.properties:2]: PR welcome imo, not critical fro this one in particular

Create issue? :)

**Review by artem-zinnatullin — APPROVED** [review]
LETS MERGE

File: gradle.properties (hunk 1 above)
Comment (raniejade) [reply gradle.properties:2]: #375

**Pull request merged**